scripts/constants.js

```diff
@@ -8,7 +8,7 @@ module.exports = Object.freeze({
   VECTOR_PRODUCTION_HOST: "vector.maps.elastic.co",
   VECTOR_STAGING_HOST: "vector-staging.maps.elastic.co",
   TILE_PRODUCTION_HOST: "tiles.maps.elastic.co",
-  TILE_STAGING_HOST: "tiles.maps.elstc.co",
+  TILE_STAGING_HOST: "tiles.maps.elastic.co",
```

Comment:
Was this a typo or an old URL that we don't need to use anymore? Just curious.

Reply:
Sorry, I forgot to mention this change in the description. This is an old URL that we don't use anymore.